runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttBindingConfig.java

```diff
@@ -226,6 +234,30 @@ private Function<String, String> asAccessor(
             });
         }
 
+        return injector;
+    }
+
+    private Function<String, String> asInjector(
+        MqttCredentialsConfig credentials)
+    {
+        Function<String, String> accessor = DEFAULT_CREDENTIALS;
+        List<MqttPatternConfig> connectPatterns = credentials.connect;
+
+        if (connectPatterns != null && !connectPatterns.isEmpty())
+        {
+            MqttPatternConfig config = connectPatterns.get(0);
+
+            accessor = orElseIfNull(accessor, connect ->
+            {
+                String result = null;
+                if (connect != null)
+                {
+                    result = config.pattern.replace("{credentials}", connect);
+                }
+                return result;
+            });
+        }
+
```

Comment:
This one should rename `accessor` to `injector`, right?
I think this was the intention of the change in `asAccessor` just not applied to this method.